### PR TITLE
libtirpc: fix #1923

### LINKED ISSUE #1923: RPCBind can't be compiled anymore
```
/usr/lib/gcc/x86_64-unknown-linux-gnu/4.9.3/../../../../lib/libtirpc.so: undefined reference to `__rpc_get_default_domain'
collect2: error: ld returned 1 exit status
Makefile:451: recipe for target 'rpcbind' failed
make: *** [rpcbind] Error 1
make: *** Waiting for unfinished jobs....
```

### PR DESCRIPTION
**Pull request by thypon**

### CODE REVIEW

File: srcpkgs/libtirpc/template [hunk 1]
```diff
@@ -1,9 +1,9 @@
 # Template file for 'libtirpc'
 pkgname=libtirpc
 version=0.3.2
-revision=1
+revision=2
 build_style=gnu-configure
-hostmakedepends="pkg-config mit-krb5-devel"
+hostmakedepends="pkg-config mit-krb5-devel automake"
```
Comment (xtraeme) [inline srcpkgs/libtirpc/template]: If don't run autoconf/automake the patch won't be applied to Makefile.am...

Comment (xtraeme) [inline srcpkgs/libtirpc/template]: i.e `pre_configure() { autoreconf -fi }`

Comment (thypon) [inline srcpkgs/libtirpc/template]: I don't think it's needed. Maybe configure script run this for me. During this phase complains if automake isn't installed, then when installed compiles cleanly (also rpcbind do the same).

Comment (beefcurtains) [inline srcpkgs/libtirpc/template]: ``configure`` should work regardless. But you'll get an error when ``make`` is invoked. Regenerating the ``configure`` script indeed does not appear to be necessary.

**Pull request merged**